### ISSUE TITLE
Hyper-Parameters of Single Domain Generalisation

### ISSUE THREAD

**Issue by guxiao0822** [BUserName/Learning_to_diversify#6]
Hi appreciate it a lot for sharing your great work! I am wondering whether it is possible to give the hyper-parameters used for single domain generalization of specific datasets. It seems like it is hardly possible to reproduce the results of PACS dataset reported in your paper if simply changing the dataloader configurations.

**Comment by xunguangwang** [comment]
Is there data fraud in this paper?

**Comment by xunguangwang** [comment]
@guxiao0822 Hi, Can you reproduce the results of PACS now?

**Comment by BUserName** [comment]
Hi @xunguangwang ,

Thanks for your interest in our work. Could you please let me know which PACS task you are working on?

For multiple source DG with resnet-18 and single DG with Alexnet, it can be reproduced the result by running
```
bash run_main_PACS.sh
```
If you are looking for resnet-18 single DG results, which were not included in the paper, you can check out the README.

Feel free to let me know if you have any further questions. You can also reach me through email.

**Comment by xunguangwang** [comment]
@BUserName Hi, I have a question. As for the results of resnet18 in the README, if the model is trained on photo domain, why acc of photo is so low?

**Comment by BUserName** [comment]
@xunguangwang  The table in README reports the single source domain generalization result. That is, in the first row, we declare which source domain we are using. And in the second row, we report the average accuracy of target domains.

For example, the 2nd column {'P': 52.29} indicates we take P as the source domain, and the average accuracy on A, C, and S is 52.29

**Comment by xunguangwang** [comment]
> @xunguangwang The table in README reports the single source domain generalization result. That is, in the first row, we declare which source domain we are using. And in the second row, we report the average accuracy of target domains.
> 
> For example, the 2nd column {'P': 52.29} indicates we take P as the source domain, and the average accuracy on A, C, and S is 52.29

Thank you very much!
